tdrs-frontend/docker-compose.local.yml

```diff
@@ -1,10 +1,12 @@
-# Local version of Docker compose, using the Dockerfile.local instead and mounting node_modules
-version: "3"
+# Local version of Docker compose, using the localdev docker build target
+# instead and additionally mounts node_modules for faster builds
+version: "3.4"
 
 services:
   tdp-frontend:
     build:
       context: .
-      dockerfile: Dockerfile.local
+      target: localdev
+    command: yarn start
```

Comment:
Since we are exiting early from the multi-stage build we need to supply this command here to get the same functionality as we had with multiple dockerfiles.